blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,164 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+export const getURLType = (url) => {
+  const imgRegex = /jpe?g|png|webp/i
+  const audioRegex = /^(https?|ftp|file):\/\/(www.)?(.*?)\.(mp3)\??(?:&?[^=&]*=[^=&]*)*$/
+  const videoRegex = /^(https?|ftp|file):\/\/(www.)?(.*?)\.(mp4|ts)\??(?:&?[^=&]*=[^=&]*)*$/
+  const uri = url.split('.')
+  let type = uri[uri.length - 1]
+  const defaultType = { label: 'Audio', type: `audio/mp3` }
+  if (new RegExp(imgRegex).test(url)) {
+    type = new RegExp(/jpe?g/i).test(type) ? 'jpeg' : type;
```

Comment:
There is a lint error on this line.  Also this line ends in with a semicolon which tells me you either are not running prettier or you're not using the local settings.  There are around 5 prettier issues in the file.